File: InsurancePointOfSaleClient/src/app/shared/VrednostAtributaOsiguranja.ts
import { TipAtributa } from "./TipAtributa";
import { Osiguranje } from "./Osiguranje";

export class VrednostAtributaOsiguranja{
    id: number;
    vrednost: string;
    grupacioniBroj: number;
    tipAtributa: TipAtributa;
    osiguranje: Osiguranje;

    constructor(vrednost: string,tipAtributa: TipAtributa){
        this.vrednost = vrednost;
        this.tipAtributa = tipAtributa;
    }
};

File: InsurancePointOfSaleService/src/main/java/com/model/VrednostAtributaOsiguranja.java
package com.model;

import java.io.Serializable;

public class VrednostAtributaOsiguranja implements Serializable{

	private static final long serialVersionUID = -7767383711555084420L;
	
	private Long id;
	private String vrednost;
	private Short grupacioniBroj;
	private TipAtributa tipAtributa;
	private Osiguranje osiguranje;
	
	public VrednostAtributaOsiguranja() {
	}

	public Long getId() {
		return id;
	}

	public void setId(Long id) {
		this.id = id;
	}

	public String getVrednost() {
		return vrednost;
	}

	public void setVrednost(String vrednost) {
		this.vrednost = vrednost;
	}

	public Short getGrupacioniBroj() {
		return grupacioniBroj;
	}

	public void setGrupacioniBroj(Short grupacioniBroj) {
		this.grupacioniBroj = grupacioniBroj;
	}

	public TipAtributa getTipAtributa() {
		return tipAtributa;
	}

	public void setTipAtributa(TipAtributa tipAtributa) {
		this.tipAtributa = tipAtributa;
	}

	public Osiguranje getOsiguranje() {
		return osiguranje;
	}

	public void setOsiguranje(Osiguranje osiguranje) {
		this.osiguranje = osiguranje;
	}
	
}
